aether-ui/aether/ui/assets/apps/utils/index.jsx

```diff
@@ -137,3 +137,21 @@ export const getLoggedInUser = () => {
  * @param {*} x -- the object
  */
 export const objectToString = (obj) => !isEmpty(obj) ? JSON.stringify(obj, 0, 2) : ''
+
+/**
+ * Generates a unique contract name per pipeline
+ *
+ * @param {Object}   pipeline
+ */
+export const generateNewContractName = (pipeline) => {
```

Comment:
This is a generic utilities file, could you move this method to a new "pipeline utilities" file? Or maybe this is not longer needed, there is no uniqueness constraint and the name can be empty till the user fills it.